schema/F24.json

```diff
@@ -0,0 +1,365 @@
+            "description": "",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 9,
+            "pattern": "^[ -~]{0,9}$",
+            "examples": [
+                "C00123456"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 2,
+                "FIELD_DESCRIPTION": "FILER COMMITTEE ID NUMBER",
+                "TYPE": "A/N-9",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "C00123456",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "this is the ID of the Committee Account the report/transaction is associated with",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "report_type": {
+            "title": "REPORT TYPE",
+            "description": "",
+            "enum":[
+                "24",
+                "48"
+            ],
+            "examples": [
+                48
+            ],
+            "fec_spec": {
+                "COL_SEQ": 3,
+                "FIELD_DESCRIPTION": "REPORT TYPE {24/48 Hour}",
+                "TYPE": "A/N-2",
+                "REQUIRED": "X(error)",
+                "SAMPLE_DATA": 48,
+                "VALUE_REFERENCE": "24, 48",
+                "RULE_REFERENCE": "Error if Code is missing;\nError if Coded incorrectly.",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "original_amendment_date": {
+            "title": "ORIGINAL AMENDMENT DATE",
+            "description": "",
+            "type": [
+                "string",
+                "null"
+            ],
+            "minLength": 0,
+            "maxLength": 8,
+            "pattern": "^\\d{0,8}$",
+            "examples": [
+                20120729
+            ],
+            "fec_spec": {
+                "COL_SEQ": 4,
+                "FIELD_DESCRIPTION": "ORIGINAL AMENDMENT DATE",
+                "TYPE": "NUM-8",
+                "REQUIRED": "X (conditional error)",
+                "SAMPLE_DATA": 20120729,
+                "VALUE_REFERENCE": "YYYYMMDD",
+                "RULE_REFERENCE": "Use date of original report or of most recent amendment.",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "committee_name": {
+            "title": "COMMITTEE NAME",
+            "description": "",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 200,
+            "pattern": "^[ -~]{0,200}$",
+            "fec_spec": {
+                "COL_SEQ": 5,
+                "FIELD_DESCRIPTION": "COMMITTEE NAME",
+                "TYPE": "A/N-200",
+                "REQUIRED": "X (warning)",
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "street_1": {
+            "title": "STREET 1",
+            "description": "",
+            "type": "string",
```

Comment:
As a "X (warning)" field is it required? If not, this "type" needs to be:

"type": [
                "string",
                "null"
            ],